ambiata-anemone.cabal

```diff
@@ -35,9 +34,6 @@ library
                        src
 
   exposed-modules:
-                       Anemone
-                       Anemone.Headers
-
```

Comment:
I removed the header file-embed - we weren't using it and it made compilation quite a bit slower.